### PR TITLE
Translating CSS chapter to Spanish

### PR DESCRIPTION
**Pull request by c-torres**
Progress on #498.

### CODE REVIEW

File: src/content/es/2019/css.md [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+
+CSS tiene este increíble poder de interpolación que se puede usar simplemente escribiendo una sola regla sobre cómo hacer la transición de esos valores. Si está utilizando CSS para administrar estados en su aplicación, ¿con qué frecuencia emplea transiciones para realizar la tarea? ¡Vamos a consultar la web!
+
+<figure>
+  <a href="/static/images/2019/02_CSS/fig25.png">
+    <img src="/static/images/2019/02_CSS/fig25.png" alt="Figura 25. Distribución del número de transiciones por página." aria-labelledby="fig25-caption" aria-describedby="fig25-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQO5CabwLwQ5Lj1_9bbEFnFM1qEqCorymaBHrcaNiMSJ7sYDKHUI5iish5VAS-SxN447UTW-1-5-OjE/pubchart?oid=419145172&amp;format=interactive">
+  </a>
+  <div id="fig25-description" class="visually-hidden">Bar chart showing the distribution of transitions per page. For desktop pages the 10, 25, 50, 75, and 90th percentiles are: 0, 2, 16, 49, and 118 transitions. The desktop distribution is much lower than mobile, by as many as 77 transitions at the 90th percentile.</div>
```
Comment (bazzadp) [inline src/content/es/2019/css.md]: Missed translation of the description

File: src/content/es/2019/css.md [hunk 2]
```diff
@@ -0,0 +1,708 @@
...
+
+<figure>
+  <a href="/static/images/2019/02_CSS/fig26.png">
+    <img src="/static/images/2019/02_CSS/fig26.png" alt="Figura 26. Distribución del número de keyframe por página." aria-labelledby="fig26-caption" aria-describedby="fig26-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQO5CabwLwQ5Lj1_9bbEFnFM1qEqCorymaBHrcaNiMSJ7sYDKHUI5iish5VAS-SxN447UTW-1-5-OjE/pubchart?oid=11848448&amp;format=interactive">
+  </a>
+  <div id="fig26-description" class="visually-hidden">Bar chart showing the distribution of keyframes per page. For mobile pages the 10, 25, 50, 75, and 90th percentiles are: 0, 0, 3, 18, and 76 keyframes. The mobile distribution is slightly higher than desktop by 6 keyframes at the 75th and 90th percentiles.</div>
```
Comment (bazzadp) [inline src/content/es/2019/css.md]: Missed translation of the description

**Comment by bazzadp** [comment]
Any update on missing translations @c-torres ?

@JMPerez / @taytus have one of you had a chance to review this yet?

File: src/content/es/2019/css.md (hunk 1 above)
Comment (c-torres) [reply src/content/es/2019/css.md]: Translation was added.

File: src/content/es/2019/css.md (hunk 2 above)
Comment (c-torres) [reply src/content/es/2019/css.md]: Translation was added.

**Comment by c-torres** [comment]
@bazzadp https://github.com/HTTPArchive/almanac.httparchive.org/pull/591/commits/8592928bcc838ca97a323b86102ac904bb1744f5 should add the missed translations, thanks for seeing them!

**Comment by bazzadp** [comment]
Friendly ping @JMPerez and/or @taytus !

**Comment by rviscomi** [comment]
In lieu of a review by Spanish speakers I think this is ok to merge to keep things moving. Any lingering translation issues can be resolved in a follow-up PR.

**Pull request merged**